plugins/net_plugin/net_plugin.cpp

```diff
@@ -122,16 +123,16 @@ namespace eos {
         sync_received(),
         sync_requested(),
         socket( s ),
-        pending_message_size(),
-        pending_message_buffer(),
+        pending_message_size(0),
+        pending_message_buffer(1024*1024*4),
+        send_buffer(1024*1024*4),
         remote_node_id(),
         last_handshake(),
         out_queue(),
         connecting (false),
-        syncing (false),
```

Comment:
Why did you delete this initializer?

Reply:
That looks like a merge problem.  I managed to retain one of them but not the other.